install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -12,262 +12,728 @@ toc::[]
-containers in your pod. Additionally, volumes which support SELinux management
-will be relabeled so that they are accessible by the specified label and,
-depending on how exclusionary the label is, only that label.
-
-This means two things:
-
-* If the container is unprivileged the volume will be given a `*type*` which is
-accessible by unprivileged containers. Usually *svirt_sandbox_file_t*.
-* If a `*level*` is specified, the volume will be labeled with the given MCS
-label.
+[[user-id]]
+== User IDs
 
 [NOTE]
 ====
-Level and MCS label are used interchangeably in this topic.
+Read link:#sccs-defaults-allowed-ranges[SCCs, Defaults, and Allowed Ranges]
+before working with supplemental groups.
 ====
 
-For your volume to be accessible by your pod, the pod must have both categories
-of the volume. So a pod with *s0:c1,c2* will be able to access volumes with
-*s0,c1,c2*, and a volume with *s0* will be accessible by all pods.
-
-[WARNING]
+[TIP]
 ====
-Hard coding MCS labels into your pod definition makes it easy for others to
-determine what MCS label is needed to access the same volume as the defined pod.
-So it is especially important to rely on the MCS labels allocated by OpenShift
-and not use this option with care.
+It is generally preferable to use group IDs
+(link:#supplemental-groups[supplemental] or link:#fsgroup[fsGroup]) to gain
+access to persistent storage versus using user IDs.
 ====
 
-SELinux options are specified as follows:
+User IDs can be defined in the container image or in the pod spec. In the pod
+definition, a single user ID can be defined globally to all containers, or
+specific to individual containers (or both). A user ID is supplied as shown in
+the pod definition fragment below:
 
+[[pod-user-id-99]]
 ====
 [source,yaml]
 ----
-apiVersion: v1
-kind: Pod
-metadata:
-  name: ebs-web
 spec:
   containers:
-    - name: web
-      image: nginx
-      ports:
-        - name: web
-          containerPort: 80
-      volumeMounts:
-          - name: ebs-volume
-            mountPath: "/usr/share/nginx/html"
-  securityContext:
-    seLinuxOptions:
-    level: "s0:c123,c456"
-  volumes:
-    - name: ebs-volume
-      awsElasticBlockStore:
-      volumeID: <VOLUME ID>
+  - name: ...
+    securityContext:
+      runAsUser: 99
 ----
 ====
 
-Currently the list of volumes which support SELinux management includes:
+ID 99 in the above is container-specific and matches the owner ID on the export.
+If the NFS export's owner ID was 54321, then that number would be used in the
+pod definition. Specifying `*securityContext*` outside of the container
+definition makes the ID global to all containers in the pod.
+
+Similar to group IDs, user IDs may be validated according to policies set in the
+SCC and/or project. If the SCC's `*runAsUser*` strategy is set to *RunAsAny*,
+then any user ID defined in the pod definition or in the image is allowed. This
+means even a UID of 0 (root) is allowed!
```

Comment:
As exciting as this statement is, I suggest removing the exclamation   :grinning: